### ISSUE TITLE
Bleeding Colors

### ISSUE THREAD

**Issue by Tny5989** [m4thmatic/PetMe#3]
You need to PopStyleColors() after you are done using them to avoid leaking resources to other addons. Any addon that uses default colors will be affected by the colors you use until the change is made.

**Comment by Tny5989** [comment]
```--Pet HP
imgui.PushStyleColor(ImGuiCol_PlotHistogram, hpBarColor);
imgui.ProgressBar(pet.HPPercent / 100, { windowSize/3-10, 15 });
imgui.PopStyleColor(1);

--Pet MP
imgui.SameLine();
imgui.PushStyleColor(ImGuiCol_PlotHistogram, colors.MpBar);
imgui.ProgressBar(petmp / 100,  { windowSize/3-10, 15 });
imgui.PopStyleColor(1);

--Pet TP
imgui.SameLine();
imgui.PushStyleColor(ImGuiCol_PlotHistogram, colors.TpBar);
imgui.ProgressBar(pettp / 3000,  { windowSize/3-10, 15 }, tostring(pettp));
imgui.PopStyleColor(1);
```

For example

**Comment by m4thmatic** [comment]
Fixed as part of 1.0.0. Thanks.
